### ISSUE TITLE
missing 1 required positional argument: 'backend'

### ISSUE THREAD

**Issue by bortzmeyer** [cbrews/ignition#3]
```
>>> r = ignition.request("gemini://gemini.circumlunar.space")
Unknown exception encountered when validating ssl certificate on gemini.circumlunar.space - load_der_x509_certificate() missing 1 required positional argument: 'backend'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/stephane/tmp/ignition/ignition/__init__.py", line 138, in request
    return req.send()
  File "/home/stephane/tmp/ignition/ignition/request.py", line 78, in send
    validation_result = self.__validate_ssl_certificate(secure_socket_result)
  File "/home/stephane/tmp/ignition/ignition/request.py", line 179, in __validate_ssl_certificate
    raise err
  File "/home/stephane/tmp/ignition/ignition/request.py", line 169, in __validate_ssl_certificate
    self.__cert_store.validate_tofu_or_add(secure_socket.server_hostname, secure_socket.getpeercert(True))
  File "/home/stephane/tmp/ignition/ignition/cert_store.py", line 107, in validate_tofu_or_add
    fingerprint, expiration = self.__parse_raw_certificate(raw_certificate)
  File "/home/stephane/tmp/ignition/ignition/cert_store.py", line 127, in __parse_raw_certificate
    cert = x509.load_der_x509_certificate(raw_certificate)
TypeError: load_der_x509_certificate() missing 1 required positional argument: 'backend'

% python3 --version   
Python 3.7.3


```

**Comment by cbrews** [comment]
@bortzmeyer Thanks for reporting this.  This seems to be a version issue inconsistency with the `cryptography` package.  I'm working on a stable fix for this now but in the meantime if you need to be unblocked please update `cryptography` to a version > 3.1.

But I'll have a stable version released later today!

**Comment by cbrews** [comment]
Released version 0.1.2 with this fix.
